pkg/converter/tool/feature.go

```diff
@@ -32,6 +32,9 @@ const (
 	// into a big batch chunk, which can reduce the the size of the image
 	// and accelerate the runtime file loading.
 	FeatureBatchSize Feature = "--batch-size"
+	// The option `--encrypt` enables converting directories, tar files
+	// or OCI images into encrypted nydus blob.
+	FeatureEncrypt Feature = "--encrypt"
```

Comment:
Should we detect if the required feature `encrypt` is supported by nydus-image? Seems we only define a constant here.

```
	requiredFeatures := tool.NewFeatures(tool.FeatureTar2Rafs)
	if opt.BatchSize != "" && opt.BatchSize != "0" {
		requiredFeatures.Add(tool.FeatureBatchSize)
	}

	detectedFeatures, err := tool.DetectFeatures(builderPath, required features, tool.GetHelp)
	if err != nil {
		return nil, err
```

Reply:
done